src/ocean_emulators/models/modules/encoder.py

```diff
@@ -15,22 +15,23 @@ class PerceiverEncoder(nn.Module):
     """A perceiver-based encoder for Samudra's flattened data (a whole column of the ocean, with history).
 
     Args:
-        n_channels (int): the number of input channels (roughly:  time x variable x (surface + depths)).
+        in_channels (int): the number of input channels (roughly:  time x variable x (surface + depths)).
         patch_size (int | tuple[int, int]): the size of the patches to embed. Patches must evenly divide the input grid.
           If a tuple is supplied, then it represents the (height, width) of the patches to embed.
         embed_dim (int): size of the latent dimension.
         perceiver_depth (int): depth of the perceiver module core.
     """
 
+    # TODO(alxmrs): Implement checkpointing
```

Comment:
```suggestion
    # TODO(alxmrs): Implement gradient checkpointing
```
I assume that's what you mean?

Reply:
Yess.